sympy/integrals/tests/test_transforms.py

```diff
@@ -868,6 +1061,7 @@ def test_issue_8882():
         **{'as_meijerg': True, 'needeval': True}))
 
 
+@XFAIL
 def test_issue_7173():
```

Comment:
Why is this XFAIL now?

Reply:
Ah, I forgot to remove it. This actually tested a failure in `meijerint`, corrected by https://github.com/sympy/sympy/pull/8905, but `sinh(a*x)*cosh(a*x)` does not go through `meijerint` anymore.

I will remove the test and write a comment in the main test list where issue #7173 is tested.